src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo-link"
+        href="#"
```

Comment:
The 'href' attribute is currently set to '#'. Consider replacing it with an actual URL or path to ensure proper navigation.